src/vue/common/TradeTopBar.vue

```diff
@@ -0,0 +1,260 @@
+import Drawer from '@/vue/common/Drawer'
+import TopBar from '@/vue/common/TopBar'
+import { vueRoutes } from '@/vue-router/routes'
+import { Bus } from '@/js/helpers/event-bus'
+import { errors } from '@tokend/js-sdk'
+import { globalize } from '@/vue/filters/globalize'
+
+const EVENTS = {
+  reloadTradeData: 'reload-trade-data',
+}
+
+export default {
+  name: 'trade-top-bar',
+  components: {
+    SelectField,
+    CreateTradeOfferForm,
+    Drawer,
+    TopBar,
+  },
+  data: () => ({
+    assetPair: {
+      base: '',
+      quote: '',
+    },
+    errors,
+    selectedPair: '',
+    tradeablePairs: [],
+    formattedPairs: [],
+    isCreateBuyOfferDrawerShown: false,
+    isCreateSellOfferDrawerShown: false,
+    vueRoutes,
+  }),
+  computed: {
+    ...mapGetters([
+      vuexTypes.accountBalances,
+    ]),
+    assetPairBalances () {
+      return {
+        base: (this.accountBalances
+          .find(i => i.asset === this.assetPair.base) || {}).balance,
+        quote: (this.accountBalances
+          .find(i => i.asset === this.assetPair.quote) || {}).balance,
+      }
+    },
+  },
+  watch: {
+    selectedPair (value) {
+      const baseAsset = value.split('/')[0]
+      const quoteAsset = value.split('/')[1]
+      this.assetPair.base = baseAsset
+      this.assetPair.quote = quoteAsset
+
+      this.$router.replace({
+        name: this.$route.name,
+        query: {
+          base: baseAsset,
+          quote: quoteAsset,
+        },
+      })
+    },
+  },
+  async created () {
+    await this.loadBalances()
+    await this.loadTradablePairs()
+  },
+  methods: {
+    ...mapActions({
+      loadBalances: vuexTypes.LOAD_ACCOUNT_BALANCES_DETAILS,
+    }),
+    async loadTradablePairs () {
+      try {
+        const response = await Sdk.horizon.assetPairs.getAll()
+        this.formatTradablePairs(response.data)
+        this.setDefaultSelectedPair(response.data)
+      } catch (error) {
+        ErrorHandler.process(error)
+      }
+    },
+    formatTradablePairs (pairs) {
+      this.formattedPairs = pairs.map(item => {
+        return `${item.base}/${item.quote}`
+      })
+    },
+    setDefaultSelectedPair (pairs) {
+      const queryBase = this.$route.query.base
+      const queryQuote = this.$route.query.quote
+
+      // if the user comes on Trade page from another place with some base and
+      // quote query params inside URL, we check that they're correct
+      // (exists in system) and set the appropriate
+      if (this.isQueryParamsValid(pairs)) {
+        const pair = pairs.find((i) => {
+          return i.base === queryBase &&
+                  i.quote === queryQuote
+        })
+        this.selectedPair = `${pair.base}/${pair.quote}`
+      } else {
+        this.selectedPair = this.formattedPairs[0]
+        if (queryBase && queryQuote) {
+          // eslint-disable-next-line
+          Bus.error(globalize('trade-top-bar.error-invalid-base-quote-query-in-link', {
```

Comment:
do not use globalize within the js. especially in this case is better to modify the StatusMessage component so it could accept the interpolation params